src/cpp/common/tls_credentials_options.cc

```diff
@@ -86,16 +83,39 @@ void TlsCredentialReloadArg::set_pem_root_certs(
 }
 
 void TlsCredentialReloadArg::add_pem_key_cert_pair(
-    TlsKeyMaterialsConfig::PemKeyCertPair pem_key_cert_pair) {
+    const TlsKeyMaterialsConfig::PemKeyCertPair& pem_key_cert_pair) {
   grpc_ssl_pem_key_cert_pair* ssl_pair =
       (grpc_ssl_pem_key_cert_pair*)gpr_malloc(
           sizeof(grpc_ssl_pem_key_cert_pair));
   ssl_pair->private_key = gpr_strdup(pem_key_cert_pair.private_key.c_str());
   ssl_pair->cert_chain = gpr_strdup(pem_key_cert_pair.cert_chain.c_str());
   ::grpc_core::PemKeyCertPair c_pem_key_cert_pair =
       ::grpc_core::PemKeyCertPair(ssl_pair);
-  c_arg_->key_materials_config->add_pem_key_cert_pair(
-      std::move(c_pem_key_cert_pair));
+  c_arg_->key_materials_config->add_pem_key_cert_pair(c_pem_key_cert_pair);
+}
+
+void TlsCredentialReloadArg::set_key_materials(
```

Comment:
Please add some comments between lines for `set_key_materials` function.

Reply:
This was addressed in a previous comment (the one with "nit").

Reply:
Thanks.